### DISCUSSION TITLE
posts/localsend/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#313] [Comments]
# posts/localsend/

LocalSend is an open source application that securely transfers files between devices without the internet. It’s cross platform meaning that it’s available for Windows, Mac, Linux, iOS (iPhone, iPad), and Android devices. This is a great alternative to AirDrop or QuickSend and can send and receive files to other devices without a 3rd party service like Google Drive.

https://technotim.live/posts/localsend/

**Comment by efritze** [comment]
Great review of Localsend, Tim.
Could you also review Feem?

Feem is like Localsend, but with support for resumable file transfers. Resumable file transfers come in very handy when your large file transfers get interrupted for one reason or the other. 

Feem is also smart enough not to duplicate files it has already transferred. This saves you time and disk space.

One last major difference is that LocalSend  is a half-duplex utility, whereas Feem is full-duplex. This means that Feem allows you to send and receive files at the same time. You don't have to wait for one large file transfer to finish before sending another file, or receiving a file from another device.

Give it a try at https://feem.io and let us know if it works for you. 

Cheers.

PS: Any issues you may encounter will 90% be about firewalls. Allow TCP and UDP ports 45001.
